packages/components/src/components/dropdown/display-text.jsx

```diff
@@ -14,9 +15,16 @@ const DisplayText = ({ className, has_symbol, list, is_align_text_left, placehol
                     value={value}
                 />
             ) : (
-                <span className={classNames('dc-dropdown__display-text', className)} name={name} value={value}>
+                <Text
+                    size='xs'
+                    color='prominent'
+                    align='center'
+                    className={classNames('dc-dropdown__display-text', className)}
+                    name={name}
+                    value={value}
```

Comment:
(question) What is the purpose of `name` and `value` here?

Reply:
I am not sure about it. Mahdi was its developer. Why is it important?